doc/user/content/guides/temporal-filters.md

```diff
@@ -11,7 +11,7 @@ You can use temporal filters to perform time-windowed computations over temporal
 
 Windows can be sliding or tumbling. Sliding windows are fixed-size time intervals that you drag over temporal data, like "Show me updates from the last ten seconds". Tumbling or hopping windows are sliding windows that slide one unit at a time, like "Show me updates from the last day for each one-hour interval".
 
-Temporal filters are defined using the function [`mz_logical_timestamp`](/sql/functions/now_and_mz_logical_timestamp). For a more detailed overview, see our blog post on [temporal filters](https://materialize.com/temporal-filters/).
+Temporal filters are defined using the function [`mz_logical_timestamp`](/sql/functions/now_and_mz_logical_timestamp), which represents the logical time at which the query executes, based on the system time defined by the system on which `materialized` is installed. For a more detailed overview, see our blog post on [temporal filters](https://materialize.com/temporal-filters/).
```

Comment:
> based on the system time defined by the system on which `materialized` is installed.

I don't think this is necessarily the case. If we do not yet have an explanation for what `mz_logical_timestamp` is, or the ideas that our computations proceed through logical times (perhaps the system times, perhaps not), we should make sure we have that, and just link to it (rather than really nail the meaning of the hard concept in a quick blurb).

Reply:
"the logical time at which the query executes, based on the system time defined by the system on which materialized is installed" is how how we currently define mz_logical_timestamp: https://materialize.com/docs/sql/functions/now_and_mz_logical_timestamp/  If that's incorrect, I should update it both there and here. Do we have any internal discussion of what it is that I can look at?

Reply:
We should tidy up that text too, then. I'm not aware of an internal discussion about it. I looked through https://materialize.com/docs/overview/what-is-materialize/ for the right concept, which is "through what times does a changing collection move". It looks like we avoid nailing that down (possibly appropriate) but the conceptual framework for "changing data" is that it changes as `mz_logical_timestamp()` advances. It needn't have anything to do with "the system time defined by the system on which materialized is installed".

I think we should probably reword https://materialize.com/docs/sql/functions/now_and_mz_logical_timestamp/ when we get a chance. I'd like to help with that when we do!